lib/assessment.js

```diff
@@ -24,7 +28,11 @@ var sum = function sum() {
 // if invoking the callback for any element
 // in the array returns a truthy value
 var any = function any(array, callback, thisArg) {
-
+  var result = true;
+    array.forEach(function(currentValue index, array) {
+      result = result && callback(currentValue, index, array);
+      return result;
+    }
```

Comment:
This could work if `return result` were below the `forEach`, and if you were passing in `array[index]` as an argument to the callback.